modules/KalturaSupport/KalturaSupport.manifest.php

```diff
@@ -810,6 +810,14 @@
 				'initvalue' => 'http://www.kaltura.com/',
 				'type' => 'url'
 			),
+			'watermarkTimeout' => array(

```

Comment:
"hideTimeout"
may be a better name since you already have watermark as the plugin key, you don't need to repeat the watermark  watermark prefix.